evadb/functions/function_bootstrap_queries.py

```diff
@@ -214,6 +214,30 @@
     EvaDB_INSTALLATION_DIR
 )
 
+Upper_function_query = """CREATE FUNCTION IF NOT EXISTS UPPER
+        INPUT  (input ANYTYPE)
+        OUTPUT (output ANYTYPE)
+        IMPL '{}/functions/helpers/upper.py';
+        """.format(
+    EvaDB_INSTALLATION_DIR
+)
+
+Lower_function_query = """CREATE FUNCTION IF NOT EXISTS LOWER
+        INPUT  (input ANYTYPE)
+        OUTPUT (output ANYTYPE)
+        IMPL '{}/functions/helpers/lower.py';
+        """.format(
+    EvaDB_INSTALLATION_DIR
+)
+
+Concat_function_query = """CREATE FUNCTION IF NOT EXISTS CONCAT
+        INPUT  (input ANYTYPE)
+        OUTPUT (output ANYTYPE)
```

Comment:
Shall the output type be String? If the output type is `ANYTYPE`, will the `CREATE TABLE ` work? Say the following query:

```
CREATE TABLE AS
SELECT new_column FROM some_table,
JOIN LATERAL CONCAT(some column, "_test") AS T(new_column)
```

Reply:
You are right, I forgot to update this.